HACKING.md

```diff
@@ -17,6 +17,30 @@ On Debian/Ubuntu:
 
     $ sudo apt install nodejs
 
+
+For running the browser unit tests, the following dependencies are required:
+
+    $ sudo apt-get install chromium-headless dbus
```

Comment:
How did you get chromium-headless installed on Ubuntu 22.04 it does not exists:

```
[jelle@⬢ ubuntu-toolbox-22.04][~]%sudo apt-get install chromium-headless dbus
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package chromium-headless
```

Reply:
Ah forgot to change it, we can run chromium using the --headless flag. Should I add that in or just remove it?